L1Trigger/L1THGCal/src/be_algorithms/HGCalClusteringImpl.cc

```diff
@@ -74,33 +74,28 @@ void HGCalClusteringImpl::clusterizeDR( const std::vector<edm::Ptr<l1t::HGCalTri
            ToBeFixed: if a tc is not a seed, but could be potencially be part of a cluster generated by a late seed, 
                       the tc will not be clusterized  
         */
-        int iclu=0;
-        vector<int> tcPertinentClusters; 
-        for( const auto& clu : clustersTmp){
-            if( this->isPertinent(**tc, clu, dr_) ){
-                tcPertinentClusters.push_back(iclu);
-            }
-            ++iclu;
-        }
-        if( tcPertinentClusters.empty() && isSeed[itc] ){
-            clustersTmp.emplace_back( *tc );
-        }
-        else if ( !tcPertinentClusters.empty() ){
-         
-            unsigned minDist(300);
-            unsigned targetClu(0);
-                        
-            for( int iclu : tcPertinentClusters){
-                double d = clustersTmp.at(iclu).distance(**tc);
-                if( d < minDist ){
-                    minDist = d;
-                    targetClu = iclu;
-                }
-            } 
 
-            clustersTmp.at(targetClu).addConstituent( *tc );                    
+	HGCalDetId tcDetId( (**tc).detId() );
```

Comment:
Indentations to be fixed. Keep things consistent with the original file: no tab, 4 spaces indent.